dist/prototypes.py

```diff
@@ -23,7 +23,7 @@
 # instead of in src/include/. As a result all relevant code is contained to a single folder. 
 # Adding modules to this list will automatically generate those headers, and we expect this list to 
 # grow as we modularise the code base.
-SELF_CONTAINED_MODULES = ["log", "evict"]
+SELF_CONTAINED_MODULES = ["log", "evict", "reconcile"]
```

Comment:
We can make this list alphabetical as part of this PR.

Reply:
Yes, done. Thanks.